droid-hal-device.inc

```diff
@@ -656,7 +657,6 @@ _remove_cruft   "/fstab.*"          \
                 "/seapp_contexts"   \
                 "/charger"          \
                 "/res"              \
-                "/data"             \
```

Comment:
In fact, I'm not sure that we need to remove the /data cleanup before it creation. Theoretically there can be some garbage.

Reply:
I would just remove it just to be on the safe side.

Reply:
Done